main.go

```diff
@@ -1,4 +1,38 @@
 package main
 
+import (
+	"github.com/kataras/iris"
+	"sample-comment-server/controllers"
+)
+
+// CommentBook is responsible of the webserver application framework container.
+// While Iris is responsible to manage the state of the application.
+// It contains and handles all the necessary parts to create a fast web server.
+type CommentBook struct {
```

Comment:
@breakingrobot Could you explain why you need to create an "container-like" struct, what are the benefits of this ?